rfcs/text/0000-Managed-Access-Data.md

```diff
@@ -0,0 +1,346 @@
+### **DCP PR:**
+
+*Leave this blank until the RFC is approved then the Author(s) must
+create a link between the assigned RFC number and this pull request in
+the format:*
+
+[dcp-community/rfc\#](https://github.com/HumanCellAtlas/dcp-community/pull/\<PR\#\>)
+
+**Informational RFC: Managed Access Data in the Datas Control Platform**
+========================================================================
+
+**Summary**
+-----------
+
+This RFC describes the requirements for the DCP to support storing and
+controlling access to managed access data.
+ 
+**Author(s)**
+-------------
+
+*Recommended format for Authors:*
```

Comment:
Lines like these can simply be deleted since they are instructions for completing the template.

Reply:
thanks